### PR TITLE
Adds elastic textarea on comments and corrects spacing

### LINKED ISSUE #127: Wrong padding on new comment form
The padding should be the same as comment card

screen shot 2015-02-26 at 16 31 22

### PR DESCRIPTION
**Pull request by joaojusto**
Fixes #127

### CODE REVIEW

File: app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim [hunk 1]
```diff
@@ -1,10 +1,15 @@
 .CardList layout="column"
   md-card.Comment id="newComment"
-    textarea#js-comment-content.Editor placeholder="State your arguments" ng-model="ctrl.comment.body"
+    textarea.Editor.Editor--card msd-elastic="\n" placeholder="State your arguments" ng-model="ctrl.comment.body"
 
-    md-toolbar md-theme="white" on-scroll="" on-scroll-anchor="#js-comment-content" scrolling-class="md-whiteframe-z1" layout="row" layout-align="end center"
-      council-raised-button.Button.Button--secondary disabled="ctrl.disabled" click="ctrl.create()" ng-hide="ctrl.comment.id"
-        | Add Comment
+    .ActionBar
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim]: What is this component?

File: app/assets/stylesheets/components/_editor.css.sass [hunk 2]
```diff
@@ -4,5 +4,9 @@
   border: none
   padding: $default-spacing * 2 $default-spacing
 
+  &.Editor--card
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_editor.css.sass]: This name doesn't seem right.

File: app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim (hunk 1 above)
Comment (joaojusto) [inline app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim]: This component purpose is to wrap all the action buttons on the comment card, like edit, comment, update and so on.

File: app/assets/stylesheets/components/_editor.css.sass (hunk 2 above)
Comment (joaojusto) [inline app/assets/stylesheets/components/_editor.css.sass]: What would you suggest then?

File: app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
 
-    md-toolbar md-theme="white" on-scroll="" on-scroll-anchor="#js-comment-content" scrolling-class="md-whiteframe-z1" layout="row" layout-align="end center"
-      council-raised-button.Button.Button--secondary disabled="ctrl.disabled" click="ctrl.create()" ng-hide="ctrl.comment.id"
-        | Add Comment
+    .CommentEditor-buttons
+      .CommentEditor-button
 
-      council-raised-button.Button.Button--secondary disabled="ctrl.disabled" click="ctrl.update()" ng-show="ctrl.comment.id"
-        | Update Comment
+      .CommentEditor-button
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim]: It's not clear the purpose of this class

**Pull request closed without merging**